### ISSUE TITLE
`anonymous-formation` complains about attributes defined in nested, named formations

### ISSUE THREAD

**Issue by h1alexbel** [objectionary/lints#562]
Check this snippet:

```eo
# Foo.
[] > foo
  start > @
    []
      build 1 > @
      [i] > build
        i > @
        ^.build (1.plus @) > next
```

Here, `build` is accessed inside the nested named formation `build`, inside the anonymous formation. However, the lint complains:

```text
It is not recommended to have anonymous formation accessing undefined object "build" inside the formation (anonymous-formation (Single))
```

We should not complain about it, since the attribute is defined and used inside the nested named formation, not at the upper level of unnamed one.

**Comment by h1alexbel** [comment]
@yegor256 WDYT?

**Comment by yegor256** [comment]
@h1alexbel indeed a bug, thanks

**Comment by 0crat** [comment]
@h1alexbel Hello! Thank you for reporting a new bug in our project. As per our team's policy, you've earned **12 points** for this valuable contribution. Your dedication to improving our project's quality is greatly appreciated. If you notice anything else that [doesn't look](https://www.yegor256.com/2018/02/06/where-to-find-more-bugs.html) right, please [don't hesitate](https://www.yegor256.com/2014/04/13/bugs-are-welcome.html) to [report](https://www.yegor256.com/2018/04/24/right-way-to-report-bugs.html) it. Your current balance stands at [+277](https://www.eolang.org/zerocracy/objectionary-vitals.html). Keep up the great work!

**Comment by h1alexbel** [comment]
the issue was fixed in the latest release: `0.0.49`
